enterprise/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -149,48 +149,55 @@ msgid ""
 "{ec_name}"
 msgstr ""
 
-#: enterprise/admin/views.py:173
+#: enterprise/admin/views.py:177
 #, python-brace-format
 msgid "Error at line {line}: {message}\n"
 msgstr ""
 
-#: enterprise/admin/views.py:200
+#: enterprise/admin/views.py:204
 #, python-brace-format
 msgid "{count} new user was linked to {enterprise_customer_name}."
 msgid_plural "{count} new users were linked to {enterprise_customer_name}."
 msgstr[0] ""
 msgstr[1] ""
 
-#: enterprise/admin/views.py:207
+#: enterprise/admin/views.py:217
 #, python-brace-format
 msgid ""
 "Some users were already linked to this Enterprise Customer: {list_of_emails}"
 msgstr ""
 
-#: enterprise/admin/views.py:214
+#: enterprise/admin/views.py:225
+#, python-brace-format
+msgid ""
+"Some users were already linked to another Enterprise Customer and were not "
+"added: {list_of_emails}"
```

Comment:
The following learners are already associated with another enterprise customer. These learners were not added to {ec_name}.